### PR TITLE
VideoTexture: Set format to RGBFormat by default.

### PR DESCRIPTION
**Pull request by mrdoob**
`Texture` sets `format` to `RGBAFormat` if the user doesn't set it.
Would it make sense to set `VideoTexture`'s `format` to `RGBFormat` by default instead?

### CODE REVIEW

**Comment by Mugen87** [comment]
I would say yes. All examples using `VideoTexture` right now set `format` to `RGBFormat`.

BTW: Would it also make sense to set `minFilter` to `LinearFilter`? Instead of this pattern:

```js
var texture = new THREE.VideoTexture( video );
texture.minFilter = THREE.LinearFilter;
texture.format = THREE.RGBFormat;
```
it would be just
```js
var texture = new THREE.VideoTexture( video );
```

File: src/textures/VideoTexture.js [hunk 1]
```diff
@@ -2,10 +2,13 @@
  * @author mrdoob / http://mrdoob.com/
  */
 
+import { RGBFormat } from '../constants.js';
 import { Texture } from './Texture.js';
 
 function VideoTexture( video, mapping, wrapS, wrapT, magFilter, minFilter, format, type, anisotropy ) {
 
+	if ( format === undefined ) format = RGBFormat;
+
```
Comment (WestLangley) [inline src/textures/VideoTexture.js]: Although we usually use this pattern:
```javascript
Texture.call( this, video, mapping, wrapS, wrapT, magFilter, minFilter, format, type, anisotropy ); 

this.format = format !== undefined ? format : RGBFormat;

this.minFilter = minFilter !== undefined ? minFilter : THREE.LinearFilter;
this.magFilter = magFilter !== undefined ? magFilter : THREE.LinearFilter;

this.generateMipmaps = false;
```

Comment (mrdoob) [reply src/textures/VideoTexture.js]: Implemented. Thanks!

**Pull request merged**